contracts/multisig/src/error.rs

```diff
@@ -28,6 +28,9 @@ pub enum ContractError {
     #[error("invalid public key format: {reason:?}")]
     InvalidPublicKeyFormat { reason: String },
 
+    #[error("public key is already registered")]
+    DuplicatePublicKey,
```

Comment:
Might be useful to return the worker address that registered the public key originally